repository_service_tuf_api/targets.py

```diff
@@ -17,7 +17,7 @@
 
 
 class ResponseData(BaseModel):
-    targets: List[str]
+    targets: Optional[List[str]]
```

Comment:
I see why you make this list optional, but it shouldn't be optional for `add/remove targets`.
Should we use other structures for those methods?
Another option will be to not do anything if this list is `None`, but if that's the case we should make sure we test it.

Reply:
You're right, actually I don't need do it. FastAPI does it.
I can just send a empty list in `publish_targets`.